README.Rmd

```diff
@@ -72,6 +72,23 @@ You can install the development version from [GitHub](https://github.com/) with:
 devtools::install_github("trytoolchest/toolchest-client-r")
 library(toolchest)
 ```
+
+### Python requirements
+The R client requires a version of **Python 3.6 or greater** with 
+`libpython` libraries, accessible via the `$PATH`. 
+
+Most manually installed versions of Python will have the necessary libraries
+installed with Python itself.
+
+For MacOS: The built-in `python` version may not have the necessary libraries.
```

Comment:
Specifying version number (or even just "recent version") may be helpful, so that people don't unnecessarily try to install Python via brew.